src/tests/test_integration.py

```diff
@@ -21,7 +21,7 @@
 __all__ = ()
 
 
-class TestEnv(object):
+class FakeTestEnv(object):
```

Comment:
So I'm happy to pull d99705e into a separate pull request. I just noticed it and fixed it while working on this.